pkg/domain/entities/images.go

```diff
@@ -54,7 +54,7 @@ type ImageSummary struct {
 	ID          string            `json:"Id"`
 	ParentId    string            `json:",omitempty"` // nolint
 	RepoTags    []string          `json:",omitempty"`
-	Created     int64             `json:",omitempty"`
+	Created     int64             `json:`
```

Comment:
Remove the `json:` tag entirely please

Reply:
done in another commit. Please let me know in case I should squash it